### ISSUE TITLE
md-fab-speed-dial (0.11.2) does not work on Windows 10 with touch events

### ISSUE THREAD

**Issue by hanthomas** [angular/material#4981]
The FAB Speed Dial demo for 0.11.2 (and all previous versions) does not work when you touch the FAB button, though it works fine with mouse.  When you touch the FAB button, it turns dark, but the speed dial is not triggered.  This problem appears to be only on Windows 10.  Android and iOS works fine.

**Comment by topherfangio** [comment]
So are you saying that it does not work on Windows mobile? Any more specifics about the version would be really helpful.

Thanks!

**Comment by hanthomas** [comment]
It's not working on *desktop* Windows 10 (not mobile or WP).  I'm running Chrome browser.  With the mouse, the demo works fine, but with touch it doesn't work at all.  

Also, while the *demo* works fine on Android and iOS devices, when I implement FAB speed dial in my code, it doesn't work at all.  I'll put together a plunkr to show what I mean.

**Comment by randallmeeker** [comment]
I can verify on the demo using a touch screen desktop version of Windows 10 on Chrome.

The first demo of Fab Speed Dial "Basic Usage" just turns black. 

The second demo "More Options" will open as expected, but will not close on touch. It will however, close on blur.

**Comment by randallmeeker** [comment]
On Windows 10, Microsoft Edge, Touch Screen

The top Demo "Basic Usage" works as expected. 

The bottom Demo "More Options" will open and then immediately close.

**Comment by randallmeeker** [comment]
@topherfangio you might add the `browser: Chrome` label or remove all the browser labels as the only common element here is Windows 10, not the browser.

**Comment by topherfangio** [comment]
@randallmeeker Any chance you can test Firefox on your touch device and see if it also has issues?

**Comment by randallmeeker** [comment]
FireFox:

The top Demo "Basic Usage" works as expected.

The bottom demo "More Options" - first click will open the tooltip, the second click will open the additional icons. Third click closes the additional icons, but the tooltip will stay until blurred. 

So basically it fails in all browsers.

**Comment by randallmeeker** [comment]
FireFox: not sure what the expected result is, but firefox will not collapse the top "Basic Usage" when blurred.

**Comment by hanthomas** [comment]
I created PR #5012 to fix this problem.  This change allows FAB speed dial to work properly on touch-only devices, mouse-only devices, and touch+mouse devices.

**Comment by topherfangio** [comment]
@hanthomas Thanks! I'll take a look!

**Comment by hanthomas** [comment]
@topherfangio I made one more change to PR #5012 to address issues on IE/Edge.  Now it should work on Chrome and IE/Edge.  Also, I verified that the latest change works on Phonegap for Android, iOS, and Windows.

**Comment by topherfangio** [comment]
@hanthomas Sorry for the delay, one of the other devs made a bunch of changes to the underlying gesture code and then I updated the speed dial to take advantage of it which drastically simplified the code.

Can you re-test the latest HEAD code and see if it you have any issues on Windows 10 touch devices?

**Comment by hanthomas** [comment]
@topherfangio - No problem. Using v1.0.0-rc2-master-c99541f, I was able to verify that FAB speed dial responds properly to both touch and mouse on the following platforms:

IE on Windows 10
Edge on Windows 10
Chrome on Windows 10
Safari on iPad 
Chrome on Android

Thanks for the fix!  I think we can close this issue along with the PR I submitted.

The only anomaly I noticed was that when I navigate to the page, it briefly shows the speed dial expanded, and then quickly collapses.  It's a minor visual annoyance and unrelated to the current issue, but I thought I'd report it.
